From a4879c0c25e3a3ab0d68cf12fea5697f0a3e1e70 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
Date: Mon, 22 Jun 2015 17:02:05 +0000
Subject: [PATCH] Fix minitest version

Signed-off-by: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
---
 rubygem-minitest_tu_shim.gemspec | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/rubygem-minitest_tu_shim.gemspec b/rubygem-minitest_tu_shim.gemspec
index ef51f42..b869dd9 100644
--- a/rubygem-minitest_tu_shim.gemspec
+++ b/rubygem-minitest_tu_shim.gemspec
@@ -27,16 +27,16 @@ Gem::Specification.new do |s|
     s.specification_version = 3
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<minitest>, ["~> 4.0"])
+      s.add_runtime_dependency(%q<minitest>, [">= 4.0"])
       s.add_development_dependency(%q<rdoc>, ["~> 4.0"])
       s.add_development_dependency(%q<hoe>, ["~> 3.7"])
     else
-      s.add_dependency(%q<minitest>, ["~> 4.0"])
+      s.add_dependency(%q<minitest>, [">= 4.0"])
       s.add_dependency(%q<rdoc>, ["~> 4.0"])
       s.add_dependency(%q<hoe>, ["~> 3.7"])
     end
   else
-    s.add_dependency(%q<minitest>, ["~> 4.0"])
+    s.add_dependency(%q<minitest>, [">= 4.0"])
     s.add_dependency(%q<rdoc>, ["~> 4.0"])
     s.add_dependency(%q<hoe>, ["~> 3.7"])
   end
-- 
2.4.2

