From d7b541e9cece16d44fb0ebac127d02d2bdce8e33 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez Gonzalez <athenas.jimenez.gonzalez@intel.com>
Date: Wed, 30 Mar 2016 16:42:47 +0000
Subject: [PATCH] Fix minitest version

---
 rubygem-minitest_tu_shim.gemspec | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/rubygem-minitest_tu_shim.gemspec b/rubygem-minitest_tu_shim.gemspec
index 3fb18b5..65dc1b6 100644
--- a/rubygem-minitest_tu_shim.gemspec
+++ b/rubygem-minitest_tu_shim.gemspec
@@ -23,16 +23,16 @@ Gem::Specification.new do |s|
   if s.respond_to? :specification_version then
     s.specification_version = 3
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<minitest>.freeze, ["~> 4.0"])
+      s.add_runtime_dependency(%q<minitest>.freeze, [">= 4.0"])
       s.add_development_dependency(%q<rdoc>.freeze, ["~> 4.0"])
       s.add_development_dependency(%q<hoe>.freeze, ["~> 3.7"])
     else
-      s.add_dependency(%q<minitest>.freeze, ["~> 4.0"])
+      s.add_dependency(%q<minitest>.freeze, [">= 4.0"])
       s.add_dependency(%q<rdoc>.freeze, ["~> 4.0"])
       s.add_dependency(%q<hoe>.freeze, ["~> 3.7"])
     end
   else
-    s.add_dependency(%q<minitest>.freeze, ["~> 4.0"])
+    s.add_dependency(%q<minitest>.freeze, [">= 4.0"])
     s.add_dependency(%q<rdoc>.freeze, ["~> 4.0"])
     s.add_dependency(%q<hoe>.freeze, ["~> 3.7"])
   end
-- 
2.7.4

